File: HelpSoftware/Help/Help.Web/Web/src/app/EF/LoginHistory.ts
import { User } from "./EF";

export class LoginHistory {
  ID_LoginHistory: Number;
  LoggedInUser: User;
  LoggedInTime: Date;
  LoginType: Number;

  deserialize(input) {
    this.ID_LoginHistory = input.ID_LoginHistory;
    this.LoggedInUser = new User().deserialize(input.LoggedInUser);
    this.LoggedInTime = input.LoggedInTime;
    this.LoginType = input.LoginType;
  }
}

File: HelpSoftware/Help/EF/LoginHistory.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Help
{
  public class LoginHistory
  {
    [Key]
    public int ID_LoginHistory { get; set; }

    public User LoggedInUser { get; set; }

    public DateTime LoggedInTime { get; set; }

    public int LoginType { get; set; }
  }
}